appcues/src/main/java/com/appcues/ui/ExperienceRenderer.kt

```diff
@@ -149,11 +206,12 @@ internal class ExperienceRenderer(
     }
 
     fun stop() {
-        renderContextManager.stop()
+        stateMachine?.stop() // modal
```

Comment:
do we really need to have a reference of the modal context? cant we just grab it from the StateMachineDirectory?

Reply:
good question - its already in the directory so it's probably sufficient to just stop all those, will double check that